src/library/autodj/autodjprocessor.cpp

```diff
@@ -174,13 +178,21 @@ AutoDJProcessor::AutoDJProcessor(
             ConfigKey(kConfigKey, kTransitionModePreferenceName),
             static_cast<int>(TransitionMode::FullIntroOutro));
     m_transitionMode = static_cast<TransitionMode>(configuredTransitionMode);
+    // TODO(c3n7) Check this out later
```

Comment:
That looks good.
Do you know if it is possible to use the getValue template with enums?

Reply:
Done. As of commit 2360eff041e876b2f17080991b3b2eb9c95f0994 I am using getValue and casting the returned value to the `FaderMode` enum.

Reply:
I need your advice on one thing. I have declared the `FaderMode` enum twice, in both `autodjprocessor.h` and `dlgprefautodj.h`. afaik repetition is never a good thing so, do I delete the declaration in `dlgprefautodj.h` and simply use integer values in `dlgprefautodj.cpp`, and if not, what is the better way to get rid of this repetition?